src/java/com/blackNecoProject/tests/ValidLoginTest.java

```diff
@@ -0,0 +1,22 @@
+package com.blackNecoProject.tests;
+
+import com.blackNecoProject.pages.LoginPage;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class ValidLoginTest extends BaseTest{
```

Comment:
Page not need because you work on login page, this test should be done on Login page